Components/Task.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { View, TouchableHighlight, Text } from 'react-native';
+import { CheckBox } from 'react-native-elements';
+
+export default class Task extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isDone: false,
+            textStyle: {
```

Comment:
don't put textStyle in state. just calculate in in place where it's needed
   <CheckBox textStyle={this.state.isDone ? styleSheet.SomeStyle: stylesheet.otherstyle}